feature-pack/src/main/resources/modules/system/layers/base/org/jboss/resteasy/resteasy-crypto/main/module.xml

```diff
@@ -38,7 +38,7 @@
         <module name="org.apache.james.mime4j"/>
         <module name="javax.servlet.api"/>
         <module name="javax.ws.rs.api"/>
-        <module name="org.bouncycastle"/>
+        <module name="org.bouncycastle" export="true"/>
```

Comment:
Hi Stuart, because BC is a 'must have' for customers to use resteasy-crypto. So in RESTEasy 'export="true"' is the default setting:

https://github.com/resteasy/Resteasy/blob/master/jaxrs/jboss-modules/src/main/resources/modules/org/jboss/resteasy/resteasy-crypto/main/module.xml#L37